src/preferences/PreferencesManager.js

```diff
@@ -528,7 +532,7 @@ define(function (require, exports, module) {
      * @return {boolean} true if a value was set
      */
     function setValueAndSave(id, value, options) {
-        DeprecationWarning.deprecationWarning("setValueAndSave called for " + id + ". Use set instead.");
```

Comment:
@le717 Wording looks OK to me. Should we make use of the second param, `oncePerCaller`, for `deprecationWarning`.